pnpbridge/src/pnpbridge/common/readme.md

```diff
@@ -0,0 +1,17 @@
+# Common helper utilities for PnP Sample
+
+This directory contains sample code that your application should be able to take with little or no modification to accelerate implementing a PnP device application.

```

Comment:
Instead of what it states currently, we should update this readme.md to state that this file contains helper routines used by the bridge to implement PnP functionality when using the Azure IoT Hub SDK.